README.md

```diff
@@ -1,6 +1,9 @@
 # junit5-system-extensions (J5SE)
 
-This project provides a set of JUnit 5 extension that allow testing behavior related to functions related to `java.lang.System` (e.g. asserting exit status codes). 
+This project provides a set of JUnit 5 extension that allow testing behavior related to functions related to `java.lang.System` (e.g. asserting exit status codes).
+
+[![Build Status](https://travis-ci.com/itsallcode/junit5-system-extensions.svg?branch=master)](https://travis-ci.com/itsallcode/junit5-system-extensions)
```

Comment:
Branch should be develop?

Reply:
I think we should change our strategy and make "master" the default branch in our repositories again. For us that is inconvenient, but it is what users expect.